test/conftest.py

```diff
@@ -0,0 +1,260 @@
+# SPDX-License-Identifier: LGPL-2.1
```

Comment:
The rest of the code uses "LGPL-2.1-only" as license identifier. Maybe it is better to keep this consistent.
Also note that this one (compared to "LGPL-2.1") also does not have a deprecation warning on spdx.org: https://spdx.org/licenses/LGPL-2.1-only.html vs https://spdx.org/licenses/LGPL-2.1.html (despite both actually state to be "LGPL-2.1 only".. )

Reply:
Fixed.